lineman/app/models/group_records_interface.coffee

```diff
@@ -6,3 +6,6 @@ angular.module('loomioApp').factory 'GroupRecordsInterface', (BaseRecordsInterfa
       @fetch
         path: "#{parentGroup.id}/subgroups"
         cacheKey: "subgroupsFor#{parentGroup.key}"
+
+    uploadPhoto: (group, file, kind) =>
+      @remote.upload("#{group.key}/upload_photo/#{kind}", file).then @remote.onSuccess, @remote.onFailure
```

Comment:
I'm pretty sure this fix wants to go into the record store itself; in this case we want to import the response into the record store, so it can update the cover and logo url automatically.